src/_guides/language/language-tour.md

```diff
@@ -2845,10 +2845,10 @@ class ProfileMark {
 ```
 
 If you want to assign the value of a `final` instance variable
```

Comment:
What do you think about changing `want` to `need`, to make it more explicit these options aren't necessarily the best choice?

```suggestion
If you need to assign the value of a `final` instance variable
```